src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo | null;
+  closeModal: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, closeModal }) => {
+  const [isLoading, setLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(selectedTodo?.id || 0)
+      .then(res => {
+        setUser(res);
+      }).catch((reg) => {
+        throw new Error(reg);
+      }).finally(() => setLoading(false));
```

Comment:
i think we can omit this
```suggestion
      .then(res => setUser(res))
      .catch((reg) => throw new Error(reg))
      .finally(() => setLoading(false));
```